src/composables/useRoutingSlip.ts

```diff
@@ -175,6 +175,44 @@ export const useRoutingSlip = () => {
     }
   }
 
+  const updateRoutingSlipRefundStatus = async (
+    status: any
+  ) => {
+    const slipNumber = routingSlip.value.number
+    try {
+      const response = await RoutingSlipService.updateRoutingSlipRefundStatus(status, slipNumber)
```

Comment:
would refactor into the service, a composable doesn't care if the status is 200, 203, it just cares if it gets the data or not

Reply:
Also think about if you were to use this call in another component, you'd be copy and pasting alot of the response handling code etc